librosa/effects.py

```diff
@@ -660,3 +661,68 @@ def preemphasis(y, coef=0.97, zi=None, return_zf=False):
         return y_out, z_f
 
     return y_out
+
+
+def deemphasis(y, coef=0.95, zi=None):
+    """De-emphasize an audio signal with the inverse operation of preemphasis():
+
+    If y = preemphasis(x, coef=coef, zi=zi), the deemphasis is:
+        x[i] -> y[i] + coef * x[i-1]
+        x = deemphasis(y, coef=coef, zi=zi)
+
+
+    Parameters
+    ----------
+    y : np.ndarray
+        Audio signal
+
+    coef : positive number
+        Pre-emphasis coefficient.  Typical values of ``coef`` are between 0 and 1.
+
+        At the limit ``coef=0``, the signal is unchanged.
+
+        At ``coef=1``, the result is the first-order difference of the signal.
+
+        The default (0.97) matches the pre-emphasis filter used in the HTK
+        implementation of MFCCs [#]_.
+
+        .. [#] http://htk.eng.cam.ac.uk/
+
+    zi : number
+        Initial filter state. If inverting a previous preemphasis(), the same value should be used.
+
+        By default ``zi`` is initialized as
+        ``((2 - coef) * y[0] - y[1]) / (3 - coef)``. This
+        value corresponds to the transformation of the default initialization of ``zi`` in ``preemphasis()``,
+        ``2*x[0] - x[1]``.
+
+    Returns
+    -------
+    y_out : np.ndarray
+        pre-emphasized signal
```

Comment:
de-emphasized signal

Reply:
Fixed in https://github.com/librosa/librosa/pull/1293/commits/c25e8c220b0e565473c16eeed82babed4ff1b1bd.